CHANGES

```diff
@@ -10,6 +10,9 @@ Incompatible changes
 * Update Underscore.js to 1.12.0
 * #6550: html: The config variable ``html_add_permalinks`` is replaced by
   :confval:`html_permalinks` and :confval:`html_permalinks_icon`
+* linkcheck: Submitting an unknown status to
+  :py:meth:`~sphinx.builders.linkcheck.CheckExternalLinksBuilder.process_result`
+  raises a :exc:`ValueError` instead of silently ignoring the unknown status.
```

Comment:
I’m not sure where this log should be located. Extensions could theoretically handle their custom status before to call the parent implementation, but this may cause a breakage.

In some way, this is a feature, it helps detect programming mistakes.

Reply:
I suppose this patch adds a sentinel condition that is never reached, right? If true, it does not affect users and developers. then no CHANGES entry is needed to me.

Reply:
Thanks for the quick answer! That’s hopefully true, the condition should never be reached.